config/routes.rb

```diff
@@ -20,6 +20,7 @@
 
   resources :status, only: [:index]
   resource :contact, only: [:show]
+  resource :privacy_policy, only: [:show]
```

Comment:
would
```ruby
resources :privacy_policy, only: [:index]
```
allow you to use `privacy_policy` everywhere? Rather than `privacy_policies`